### DISCUSSION TITLE
Multi Column Layout

### DISCUSSION THREAD

**Discussion by SKempin** [chakra-ui/chakra-ui#5741] [Q&A]
Is it possible to achieve Multi Column Layouts with Chakra? If so can I have an example? Thanks

**Comment by nikolovlazar** [comment]
Hello @SKempin! What do you mean by multi-column layouts? Like the Chakra UI docsite has, for example? Here are some examples that might help you get started:
- https://github.com/chakra-ui/chakra-ui-docs
- https://github.com/chakra-ui/design-vs-dev/tree/main/challenges/chat-app
- https://github.com/chakra-ui/design-vs-dev/tree/main/challenges/afrosonic-concept-dashboard

**Reply by SKempin** [reply]
Hi @nikolovlazar , I mean this - https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Columns

Where content is split evenly across multiple columns.

**Reply by nikolovlazar** [reply]
Right! The Chakra UI docsite has a multi-column layout. Feel free to poke around and figure out how to do it. And to answer your question, yes! Chakra UI's style props are basically CSS properties. If you can achieve something in CSS, you can also achieve it in Chakra UI.
